bl-languages/fi_FI.json

```diff
@@ -0,0 +1,242 @@
+	"use-this-field-to-name-your-site": "Kirjoita tähän sivusi nimi. Se näkyy jokaisen sivun yläreunassa.",
+	"use-this-field-to-add-a-catchy-phrase": "Voit keksiä tähän esimerkiksi mainoslauseen tms.",
+	"you-can-add-a-site-description-to-provide": "Voit lisätä lyhyen kuvauksen sivustostasi.",
+	"you-can-add-a-small-text-on-the-bottom": "Voit lisätä pienen tekstinpätkän jokaisen sivun loppuun. Se voi olla vaikkapa sivuston nimi, yhteystietoja, päivämäärä, jne.",
+	"number-of-posts-to-show-per-page": "Kuinka monta blogikirjoitusta yhdellä sivulla saa näkyä.",
+	"the-url-of-your-site": "Sivustosi internetosoite.",
+	"add-or-edit-description-tags-or": "Lisää ja muokkaa kuvausta, tageja tai sivun osoitetta.",
+	"select-your-sites-language": "Valitse sivuston kieli.",
+	"select-a-timezone-for-a-correct": "Valitse aikavyöhyke, jotta kellonajat ja päivämäärät näkyvät oikein.",
+	"you-can-use-this-field-to-define-a-set-off": "Tällä kentällä voit määritellä joukon asetuksia liittyen kieleen, maahan ja erityisiin asetuksiin.",
+	"you-can-modify-the-url-which-identifies":"Voit muokata sivun osoitetta. Enintään 150 merkkiä",
+	"this-field-can-help-describe-the-content": "Kirjoita tähän kuvaus sivun sisällöstä. Kuvaus näkyy hakutuloksessa esim. Googlessa. Enintään 150 merkkiä.",
+
+	"delete-the-user-and-all-its-posts":"Poista käyttäjä, sekä kaikki sen blogikirjoitukset",
+	"delete-the-user-and-associate-its-posts-to-admin-user": "Poista käyttäjä, ja siirrä sen blogikirjoitukset ylläpitäjän nimiin",
+	"read-more": "Lue lisää...",
+	"show-blog": "Näytä blogi",
+	"default-home-page": "Sivuston etusivu",
+	"version": "Versio",
+	"there-are-no-drafts": "Ei luonnoksia.",
+	"create-a-new-article-for-your-blog":"Kirjoita blogiisi.",
+	"create-a-new-page-for-your-website":"Luo uusi sivu sivustollesi.",
+	"invite-a-friend-to-collaborate-on-your-website":"Tee ystävällesi omat tunnukset, niin hänkin voi kirjoittaa sivustollesi",
+	"change-your-language-and-region-settings":"Muuta kielen ja alueen asetuksia.",
+	"language-and-timezone":"Kieli ja aikavyöhyke",
+	"author": "Tekijä",
+	"start-here": "Aloita tästä",
+	"install-theme": "Asenna teema",
+	"first-post": "Ensimmäinen blogikirjoitus",
+	"congratulations-you-have-successfully-installed-your-bludit": "Onneksi olkoon! **Bludit** on nyt asennettu!",
+	"whats-next": "Mitä seuraavaksi?",
+
+	"follow-bludit-on": "Seuraa Bludit:ia",
+	"visit-the-support-forum": "Vieraile [foorumilla](https://forum.bludit.com) jos tarvitse tukea",
+	"read-the-documentation-for-more-information": "[Dokumentaatiosta](https://docs.bludit.com) löydät paljon lisää tietoa",
+	"share-with-your-friends-and-enjoy": "Jaa ystäviesi kanssa ja nauti!",
+	"the-page-has-not-been-found": "Etsimääsi sivua ei löydy.",
+	"error": "Virhe",
+	"bludit-installer": "Bludit Installer",
+	"welcome-to-the-bludit-installer": "Tervetuloa asentamaan Bludit!",
```

Comment:
"welcome-to-the-bludit-installer": "Tervetuloa asentamaan Bludit-sisällönhallintajärjestelmää!",
or
"welcome-to-the-bludit-installer": "Tervetuloa asentamaan Bluditia!",